graf2d/gpad/src/TPad.cxx

```diff
@@ -6204,7 +6206,7 @@ void TPad::ShowGuidelines(TObject *object, const Int_t event, const char mode, c
    TPad *is_pad = dynamic_cast<TPad *>( object );
    TVirtualPad *padSave = 0;
    padSave = gPad;
-   if (is_pad) is_pad->GetMother()->cd();
+   if (is_pad && is_pad->GetMother()) is_pad->GetMother()->cd();
```

Comment:
In principle is_pad is evaluated before is_pad->GetMother() and calling GetMother in the if statement should be save. But logically speaking, to make it more clear, is it not better to do:
```
if (is_pad) 
   if (is_pad->GetMother()) is_pad->GetMother()->cd();
```